README.md

```diff
@@ -4,13 +4,248 @@ NATS Streaming is an extremely performant, lightweight reliable streaming platfo
+        --tlscert <file>             Server certificate file
+        --tlskey <file>              Private key for server certificate
+        --tlsverify                  Enable TLS, very client certificates
+        --tlscacert <file>           Client certificate CA for verification
+
+NATS Clustering Options:
+        --routes <rurl-1, rurl-2>    Routes to solicit and connect
+        --cluster <cluster-url>      Cluster URL for solicited routes
+
+Common Options:
+    -h, --help                       Show this message
+    -v, --version                    Show version
+        --help_tls                   TLS help.
+```
+
+## Securing NATS Streaming Server
+
+### Authorization
+
+If you use basic authorization, that is a single user defined through the configuration file or command line parameters, nothing else is required. The embedded NATS Server is started with those credentials and the NATS Streaming Server uses them.
+
+However, if you define multiple users, then you **must** specify the username and password (or token) corresponding to the NATS Streaming Server's user.
+
+For instance, suppose that your configuration file `server.cfg` has the following content:
+
+```
+listen: 127.0.0.1:4233
+http: 127.0.0.1:8233
+
+authorization {
+    users = [
+      {user: alice, password: foo}
+      {user: bob,   password: bar}
+    ]
+}
+```
+and you start the NATS Streaming Server this way:
+```sh
+nats-streaming-server -config server.cfg
+```
+then the server would fail to start. You **must** specify the user used by the streaming server. For instance:
+```sh
+nats-streaming-server -config server.cfg -user alice -pass foo
+```
+
+### TLS
+
+While there are several TLS related parameters to the streaming server, securing the NATS Streaming server's connection is straightforward when you bear in mind that the relationship between the NATS Streaming server and the embedded NATS server is a client server relationship.  To state simply, the streaming server is a client of it's embedded NATS server.
+
+That means two sets of TLS configuration parameters must be used:  TLS server parameters for the embedded NATS server, and TLS client parameters for the streaming server itself.
+
+The streaming server specifies it's TLS client certificates with the following three parameters:
+    -tls_client_key              Client key for the streaming server
+
+    -tls_client_cert             Client certificate for the streaming server
+
+    -tls_client_cacert           Client certificate CA for the streaming server
+
+These could be the same certificates used with your NATS streaming clients.
+
+The embedded NATS server specifies TLS server certificates with these:
+
+        --tlscert <file>             Server certificate file
+
+        --tlskey <file>              Private key for server certificate
+
+        --tlscacert <file>           Client certificate CA for verification
+
+The server parameters are used the same way you'd secure a typical NATS server.
```

Comment:
For "secure a typical NATS server" add link to https://github.com/nats-io/gnatsd#securing-nats.